2025-05-17 15:16:01,521 - test_factory_modeling.py:[25] - [INFO] - 本用例正在执行：test_productaterials01。获取到的结果是：数据新增成功, 期望的结果是:数据更新成功
2025-05-17 15:16:01,521 - test_factory_modeling.py:[25] - [INFO] - 本用例正在执行：test_productaterials01。获取到的结果是：数据新增成功, 期望的结果是:数据更新成功
2025-05-17 15:16:01,528 - test_factory_modeling.py:[31] - [ERROR] - 登录页面：test_login01 断言失败，展开结果与预期不符
2025-05-17 15:16:01,528 - test_factory_modeling.py:[31] - [ERROR] - 登录页面：test_login01 断言失败，展开结果与预期不符
2025-05-17 15:16:01,757 - test_login.py:[24] - [INFO] - 本用例正在执行：test_login01。获取到的结果是：数据更新成功, 期望的结果是:数据更新成功，返回token
2025-05-17 15:16:01,757 - test_login.py:[24] - [INFO] - 本用例正在执行：test_login01。获取到的结果是：数据更新成功, 期望的结果是:数据更新成功，返回token
2025-05-17 15:16:01,757 - test_login.py:[24] - [INFO] - 本用例正在执行：test_login01。获取到的结果是：数据更新成功, 期望的结果是:数据更新成功，返回token
2025-05-17 15:16:01,942 - test_login.py:[41] - [INFO] - 本用例正在执行：test_login02。获取到的结果是：密码错误, 期望的结果是:账号错误，不返回token
2025-05-17 15:16:01,942 - test_login.py:[41] - [INFO] - 本用例正在执行：test_login02。获取到的结果是：密码错误, 期望的结果是:账号错误，不返回token
2025-05-17 15:16:01,942 - test_login.py:[41] - [INFO] - 本用例正在执行：test_login02。获取到的结果是：密码错误, 期望的结果是:账号错误，不返回token
2025-05-17 15:16:01,942 - test_login.py:[41] - [INFO] - 本用例正在执行：test_login02。获取到的结果是：密码错误, 期望的结果是:账号错误，不返回token
2025-05-17 15:16:01,968 - test_login.py:[58] - [INFO] - 本用例正在执行：test_login03。获取到的结果是：用户不存在, 期望的结果是:用户不存在，不返回token
2025-05-17 15:16:01,968 - test_login.py:[58] - [INFO] - 本用例正在执行：test_login03。获取到的结果是：用户不存在, 期望的结果是:用户不存在，不返回token
2025-05-17 15:16:01,968 - test_login.py:[58] - [INFO] - 本用例正在执行：test_login03。获取到的结果是：用户不存在, 期望的结果是:用户不存在，不返回token
2025-05-17 15:16:01,968 - test_login.py:[58] - [INFO] - 本用例正在执行：test_login03。获取到的结果是：用户不存在, 期望的结果是:用户不存在，不返回token
2025-05-17 15:16:01,968 - test_login.py:[58] - [INFO] - 本用例正在执行：test_login03。获取到的结果是：用户不存在, 期望的结果是:用户不存在，不返回token
2025-05-17 15:16:02,075 - main1.py:[40] - [ERROR] - 测试执行失败，错误信息: 
2025-05-17 15:16:06,359 - main1.py:[193] - [INFO] - 钉钉发送状态码: 200
2025-05-17 15:16:06,360 - main1.py:[126] - [INFO] - 端口 8000 可用，尝试启动服务器...
2025-05-17 15:16:09,381 - main1.py:[145] - [INFO] - 端口 8000 可连接，生成报告链接...
2025-05-17 15:16:11,213 - main1.py:[172] - [INFO] - 邮件发送成功
2025-05-17 15:17:34,498 - test_login.py:[24] - [INFO] - 本用例正在执行：test_login01。获取到的结果是：数据更新成功, 期望的结果是:数据更新成功，返回token
2025-05-17 15:17:34,498 - test_login.py:[24] - [INFO] - 本用例正在执行：test_login01。获取到的结果是：数据更新成功, 期望的结果是:数据更新成功，返回token
2025-05-17 15:17:34,708 - test_login.py:[41] - [INFO] - 本用例正在执行：test_login02。获取到的结果是：密码错误, 期望的结果是:账号错误，不返回token
2025-05-17 15:17:34,708 - test_login.py:[41] - [INFO] - 本用例正在执行：test_login02。获取到的结果是：密码错误, 期望的结果是:账号错误，不返回token
2025-05-17 15:17:34,708 - test_login.py:[41] - [INFO] - 本用例正在执行：test_login02。获取到的结果是：密码错误, 期望的结果是:账号错误，不返回token
2025-05-17 15:17:34,733 - test_login.py:[58] - [INFO] - 本用例正在执行：test_login03。获取到的结果是：用户不存在, 期望的结果是:用户不存在，不返回token
2025-05-17 15:17:34,733 - test_login.py:[58] - [INFO] - 本用例正在执行：test_login03。获取到的结果是：用户不存在, 期望的结果是:用户不存在，不返回token
2025-05-17 15:17:34,733 - test_login.py:[58] - [INFO] - 本用例正在执行：test_login03。获取到的结果是：用户不存在, 期望的结果是:用户不存在，不返回token
2025-05-17 15:17:34,733 - test_login.py:[58] - [INFO] - 本用例正在执行：test_login03。获取到的结果是：用户不存在, 期望的结果是:用户不存在，不返回token
2025-05-17 15:17:38,726 - main1.py:[193] - [INFO] - 钉钉发送状态码: 200
2025-05-17 15:17:38,727 - main1.py:[126] - [INFO] - 端口 8001 可用，尝试启动服务器...
2025-05-17 15:17:41,750 - main1.py:[145] - [INFO] - 端口 8001 可连接，生成报告链接...
2025-05-17 15:17:43,311 - main1.py:[172] - [INFO] - 邮件发送成功
2025-05-17 15:22:55,341 - test_login.py:[24] - [INFO] - 本用例正在执行：test_login01。获取到的结果是：数据更新成功, 期望的结果是:数据更新成功，返回token
2025-05-17 15:22:55,341 - test_login.py:[24] - [INFO] - 本用例正在执行：test_login01。获取到的结果是：数据更新成功, 期望的结果是:数据更新成功，返回token
2025-05-17 15:22:55,512 - test_login.py:[41] - [INFO] - 本用例正在执行：test_login02。获取到的结果是：密码错误, 期望的结果是:账号错误，不返回token
2025-05-17 15:22:55,512 - test_login.py:[41] - [INFO] - 本用例正在执行：test_login02。获取到的结果是：密码错误, 期望的结果是:账号错误，不返回token
2025-05-17 15:22:55,512 - test_login.py:[41] - [INFO] - 本用例正在执行：test_login02。获取到的结果是：密码错误, 期望的结果是:账号错误，不返回token
2025-05-17 15:22:55,538 - test_login.py:[58] - [INFO] - 本用例正在执行：test_login03。获取到的结果是：用户不存在, 期望的结果是:用户不存在，不返回token
2025-05-17 15:22:55,538 - test_login.py:[58] - [INFO] - 本用例正在执行：test_login03。获取到的结果是：用户不存在, 期望的结果是:用户不存在，不返回token
2025-05-17 15:22:55,538 - test_login.py:[58] - [INFO] - 本用例正在执行：test_login03。获取到的结果是：用户不存在, 期望的结果是:用户不存在，不返回token
2025-05-17 15:22:55,538 - test_login.py:[58] - [INFO] - 本用例正在执行：test_login03。获取到的结果是：用户不存在, 期望的结果是:用户不存在，不返回token
2025-05-17 15:23:00,490 - main1.py:[193] - [INFO] - 钉钉发送状态码: 200
2025-05-17 15:23:00,491 - main1.py:[126] - [INFO] - 端口 8002 可用，尝试启动服务器...
2025-05-17 15:23:03,508 - main1.py:[145] - [INFO] - 端口 8002 可连接，生成报告链接...
2025-05-17 15:23:05,141 - main1.py:[172] - [INFO] - 邮件发送成功
2025-05-17 16:21:55,667 - test_login.py:[24] - [INFO] - 本用例正在执行：test_login01。获取到的结果是：数据更新成功, 期望的结果是:数据更新成功，返回token
2025-05-17 16:21:55,667 - test_login.py:[24] - [INFO] - 本用例正在执行：test_login01。获取到的结果是：数据更新成功, 期望的结果是:数据更新成功，返回token
2025-05-17 16:21:55,846 - test_login.py:[41] - [INFO] - 本用例正在执行：test_login02。获取到的结果是：密码错误, 期望的结果是:账号错误，不返回token
2025-05-17 16:21:55,846 - test_login.py:[41] - [INFO] - 本用例正在执行：test_login02。获取到的结果是：密码错误, 期望的结果是:账号错误，不返回token
2025-05-17 16:21:55,846 - test_login.py:[41] - [INFO] - 本用例正在执行：test_login02。获取到的结果是：密码错误, 期望的结果是:账号错误，不返回token
2025-05-17 16:21:55,876 - test_login.py:[58] - [INFO] - 本用例正在执行：test_login03。获取到的结果是：用户不存在, 期望的结果是:用户不存在，不返回token
2025-05-17 16:21:55,876 - test_login.py:[58] - [INFO] - 本用例正在执行：test_login03。获取到的结果是：用户不存在, 期望的结果是:用户不存在，不返回token
2025-05-17 16:21:55,876 - test_login.py:[58] - [INFO] - 本用例正在执行：test_login03。获取到的结果是：用户不存在, 期望的结果是:用户不存在，不返回token
2025-05-17 16:21:55,876 - test_login.py:[58] - [INFO] - 本用例正在执行：test_login03。获取到的结果是：用户不存在, 期望的结果是:用户不存在，不返回token
2025-05-17 16:22:02,536 - main1.py:[92] - [ERROR] - 发送钉钉通知失败: HTTPSConnectionPool(host='oapi.dingtalk.com', port=443): Max retries exceeded with url: /robot/send?access_token=727fef137b72ab2cabd7dfded3f55a95dbe8dc71c1bd1496f0673f3fc250d33c (Caused by ProxyError('Cannot connect to proxy.', OSError(0, 'Error')))
2025-05-17 16:22:02,543 - main1.py:[193] - [INFO] - 钉钉发送状态码: 500
2025-05-17 16:22:02,545 - main1.py:[126] - [INFO] - 端口 8003 可用，尝试启动服务器...
2025-05-17 16:22:05,576 - main1.py:[145] - [INFO] - 端口 8003 可连接，生成报告链接...
2025-05-17 16:22:06,694 - main1.py:[172] - [INFO] - 邮件发送成功
